tests/fuzz/process.py

```diff
@@ -0,0 +1,50 @@
+import os
+import argparse
+import subprocess
+from time import time
+from RLTest import Env
+from redisgraph import Graph
+from redis import ResponseError
+
+
+def make_connection():
+    env = Env(decodeResponses=True, module="../../src/redisgraph.so", logDir="logs")
+    redis_con = env.getConnection()
+    return Graph("G", redis_con)
+
+
+def issue_queries(graph, timeout):
+    working_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Style: wrap abspath around dirname instead of dirname around abspath since dirname just removes the last segment of the path. It's more stable on symlinked directories.